src/entity/SearchRequest.java

```diff
@@ -0,0 +1,25 @@
+package entity;
+
+public class SearchRequest {
+
+  private final String queryRequest;
+
+  private String queryResponse;
```

Comment:
The query response is a structured JSON object right? Can I see a sample response, and then I can help construct a better data type here?

Reply:
Also, for clean architecture you'll need to split this into two files (request and response)

Reply:
yeah, added new classes, and implementation of the data access object.